tools/PI/DevHome.PI/Telemetry/TelemetryReporter.cs

```diff
@@ -76,7 +76,7 @@ public void SwitchTo(Feature requestedFeature)
                     if (FeatureState.IsExclusive(flag))
                     {
                         // First fire any events to stop the previous features telemetry.
-                        Debug.Assert(eventGenerator != null, "eventGenerator is null");
+                        // Debug.Assert(eventGenerator != null, "eventGenerator is null");

```

Comment:
It's firing like mad right now. Had to comment it out to make forward progress. I suspect this is because it's in a checked in but off state right now (or maybe just buggy). I know @jaholme is taking a look at the telemetry story all up.